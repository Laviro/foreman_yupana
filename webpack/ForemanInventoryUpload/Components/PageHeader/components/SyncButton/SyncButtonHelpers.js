import { foremanUrl } from '../../../../../ForemanRhCloudHelpers';

export const inventorySyncUrl = path => foremanUrl(`/inventory_sync/${path}`);
