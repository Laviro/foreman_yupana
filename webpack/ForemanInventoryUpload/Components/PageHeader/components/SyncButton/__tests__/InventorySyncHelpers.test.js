import { testSelectorsSnapshotWithFixtures } from '@theforeman/test';
import { inventorySyncUrl } from '../SyncButtonHelpers';

const fixtures = {
  'should return insights cloud Url': () => inventorySyncUrl('test_path'),
};

describe('SyncButton helpers', () =>
  testSelectorsSnapshotWithFixtures(fixtures));
