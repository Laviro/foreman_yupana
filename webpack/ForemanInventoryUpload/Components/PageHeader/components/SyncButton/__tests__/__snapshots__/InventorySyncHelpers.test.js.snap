// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SyncButton helpers should return insights cloud Url 1`] = `"/inventory_sync/test_path"`;
